### ISSUE TITLE
Add 2-dimension and 3-dimension arrays to the note about pixel formats

### ISSUE THREAD

**Issue by techtonik** [drj11/pypng#17]
http://pythonhosted.org/pypng/png.html#a-note-on-formats - speaks about:
- BRFP
- FRFP
- BRBP

http://pythonhosted.org/pypng/png.html#png.from_array - speaks about:
[row structure]
- 2-dimensional array
- 3-dimensional array
  [pixel structure]
  - L (level)
  - LA (level alpha)
  - RGB
  - RGBA

The notes on formats and from_array reference can be generalized and merged. The format note talks basically about [row structure] - it does not provide any visual reference for [pixel structure] except mentioning somewhere deep that it can be RGBA, LA or fourth (!) `packed` format. When I say `visual reference` I mean separate paragraph that looks like picture - it has visual structure as a list or table - not just plain text. I find from_array reference description much more clear and up to the point.

This basically was born out of the observation that a person reading pypng docs for the first time is most likely looking for answers:
- Is pixel in pypng the same as pixel I know?
- How the pixel that I know is represented in pypng?

Luckily, the RGBA format is familiar to everybody, but what about grayscale? For example, I made my image from_array using 2DL format, but how I should modify it now? Do I have to convert my algorithm to convert everything to RGBA values?

The really good point of current documentation is that explanation is done by example. The only awesome enhancement that can be made are a few pictures to spice up the text a bit (like Scapy docs do).

To summarize, this is an enhancement proposal to describe pypng from the pixel-centric perspective. =)
